### PR TITLE
Add basic readme

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
@miltontony ready for review

File: README.rst [hunk 1]
```diff
@@ -2,3 +2,36 @@ casepropods.dummy
...
+          'items': [{
+              'name': 'EDD',
+              'value': '2015-07-18'
+          }, {
+              'name': 'Clinic Code',
+              'value': '2034 6524 6421'
+          }]
+      }
+  }, {
+      'label': 'custom_pod',
```
Comment (hodgestar) [inline README.rst]: Hmm. I'm a little worried that people won't know what `custom_pod` is. Should we perhaps leave it out or add a comment that this is a different kind of pod?

Comment (justinvdm) [inline README.rst]: Woops, copy-paste mistake, fixing.

Comment (justinvdm) [inline README.rst]: Fixed.

File: README.rst [hunk 2]
```diff
@@ -2,3 +2,33 @@ casepropods.dummy
...
+~~~~~
+
+In casepro's settings file, add the pod as an installed app, and configure the pods that you would like to show in the UI:
+
+.. code-block:: python
+
+  INSTALLED_APPS += ('casepropods.dummy.plugin.DummyPodPlugin',)
+
+  PODS = [{
+      'label': 'dummy_pod',
```
Comment (hodgestar) [inline README.rst:24]: Do you think it's clear enough that this label has to match the one on the `DummyPodPlugin` class, or should we explain that somehow? Maybe in a comment?

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**